sourcemap/cache.go

```diff
@@ -10,7 +10,10 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 )
 
-const MIN_CLEANUP_INTERVAL_SECONDS float64 = 60
+const (
+	MIN_CLEANUP_INTERVAL_SECONDS float64 = 60
+	LOGGER_SELECTOR                      = "sourcemap"
```

Comment:
Constants are usually written in [MixedCase](https://golang.org/doc/effective_go.html#mixed-caps).